components/Toolbar.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { StyleSheet, View, Dimensions, Image } from 'react-native';
+
+export default class PrimaryButton extends React.Component {
+  
+    render() {
+      return (
+           <View style={[styles.formatInBox, {marginBottom: this.props.marginBottom}, {marginTop: this.props.marginTop}]}>
+              <Image source ={require('./../assets/images/shopping-bag.png')} style={styles.bag}/>
+              <Image source ={require('./../assets/images/personicon.png')} style={styles.person}/>
+           </View>
+      );
+    }
+}
+
+const styles = StyleSheet.create({
+    formatInBox: {
+        flexDirection: 'row', 
+        justifyContent: 'space-between', 
+        width: Dimensions.get('screen').width,
+        alignItems: 'center',
+        height: 43,
+        backgroundColor: '#FF715B',
+    },
+    person:{
```

Comment:
Use the `variables` file that you created - the point of that file is that you don't have to type the same stuff over and over again